### PR TITLE
unit test cleanups

### PR DESCRIPTION
**Pull request by claws**
This pull requests implements a number of cleanups in the unit tests. This pull request does not modify the functionality of the unit tests.

Primarily it provides changes that silence a range of linter and style (e.g. PEP8) warnings. These warnings and issues present a lot of noise in development environments that run linters (e.g SublimeText using MagicPython, etc). This can be distracting for developers trying to work in the unit tests.

In summary the changes fall into the following categories:

- removes unused imports
- remove duplicated test methods (test_stdin_not_inheritable)
- too many blank lines
- no blank line 
- line length to long
- warnings for method args (incorrect indent, mis-matched indent, etc)
- no whitespace between operators (e.g. 1/0 --> 1 / 0)

### CODE REVIEW

**Comment by 1st1** [comment]
Thanks a lot!
